src/components/MapArea.tsx

```diff
@@ -23,11 +27,14 @@ const BASE_LAYERS_ARRAY = [
   {label: 'Dark', value: 'dark-v10'}
 ]
 const StyledMapArea = styled.div`
-  border: 2px solid green;
-  height: 80%;
+  // display: flex;
+  // flex-direction: row;
+  // flex-grow: 1
+  height: 78vh;
 `
 const StyledMapAreaContainer = styled.div`
   flex: 1;
+  height: 95vh;
```

Comment:
does this mean that the Map should take up 95% of the screen height?

Reply:
Correct., 95% of the viewport height. Is this bad practice? I could define it as a percentage of the parent height instead